app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -56,14 +56,15 @@ case class GlobalAttributeForAPI(val globalAttributeId: Int,
         "agree_count" -> agreeCount,
         "disagree_count" -> disagreeCount,
         "notsure_count" -> notsureCount,
-        "users" -> usersList
+        "users" -> usersList,
+        "cluster_size" -> labelCount
       )
     )
   }
   val attributesToArray = Array(globalAttributeId, labelType, streetEdgeId, osmStreetId, neighborhoodName, lat.toString,
                                 lng.toString, avgImageCaptureDate, avgLabelDate.toString,
                                 severity.getOrElse("NA").toString, temporary.toString, agreeCount.toString,
-                                disagreeCount.toString, notsureCount.toString, "\"[" + usersList.mkString(",") + "]\"")
+                                disagreeCount.toString, notsureCount.toString, "\"[" + usersList.mkString(",") + "]\"", labelCount.toString)
```

Comment:
This brings us over our max line length according to [the style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-guide). Can you have the label count on the following line?